operator/v1/types_csi_cluster_driver.go

```diff
@@ -369,6 +369,24 @@ type VSphereCSIDriverConfigSpec struct {
 	// +openshift:enable:FeatureGate=VSphereDriverConfiguration
 	// +optional
 	GranularMaxSnapshotsPerBlockVolumeInVVOL *uint32 `json:"granularMaxSnapshotsPerBlockVolumeInVVOL,omitempty"`
+
+	// maxAllowedBlockVolumesPerNode is a configuration parameter that allows setting custom value for limit of
```

Comment:
The optional marker doesn't show up in the generated documentation that users of `oc explain ...` would see so it should be called out explicitly in the Godoc that this is an optional field:
```suggestion
	// maxAllowedBlockVolumesPerNode is an optional configuration parameter that allows setting custom value for limit of
```